### PR TITLE
Security profile: Add basic structure as md-files

### PR DESCRIPTION
**Pull request by armintaenzertng**
I converted the structure found in the `security-profile.png` into model md-files.

Three issues arise:
- [ ] #165 
- [ ] #166 
- [ ] #167

### CODE REVIEW

**Comment by maxhbr** [comment]
Maybe helpful: this is a visualization generated from the markdown data on https://github.com/armintaenzertng/spdx-3-model/commit/2e8131b449bc898a819280c51a6016921e150e20:

Security Profile

File: model/Security/Classes/EpssVulnerabilityCategorization.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+# EpssVulnerabilityCategorization
+
+## Summary
+
+TODO
+
+## Description
+
+An EpssVulnerabilityCategorization is TODO
```
Comment (jeff-schutt) [inline model/Security/Classes/EpssVulnerabilityCategorization.md:11]:
```suggestion
[https://www.first.org/epss/model](https://www.first.org/epss/model)
```

File: model/Security/Classes/CvssV2VulnerabilityCategorization.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+# CvssV2VulnerabilityCategorization
+
+## Summary
+
+TODO
+
+## Description
+
+A CvssV2VulnerabilityCategorization is TODO
```
Comment (jeff-schutt) [inline model/Security/Classes/CvssV2VulnerabilityCategorization.md:11]:
```suggestion
[https://www.first.org/cvss/v2/guide](https://www.first.org/cvss/v2/guide)
```

File: model/Security/Classes/CvssV3VulnerabilityCategorization.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+# CvssV3VulnerabilityCategorization
+
+## Summary
+
+TODO
+
+## Description
+
+A CvssV3VulnerabilityCategorization is TODO
```
Comment (jeff-schutt) [inline model/Security/Classes/CvssV3VulnerabilityCategorization.md:11]:
```suggestion
[https://www.first.org/cvss/v3.1/specification-document](https://www.first.org/cvss/v3.1/specification-document)
```

File: model/Security/Classes/ExploitCatalogVulnerabilityCategorization.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# ExploitCatalogVulnerabilityCategorization
```
Comment (jeff-schutt) [inline model/Security/Classes/ExploitCatalogVulnerabilityCategorization.md:3]:
```suggestion
# KnownExploitedVulnerabilitiesCatalogCategorization
```

File: model/Security/Classes/ExploitCatalogVulnerabilityCategorization.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+
+## Summary
+
+TODO
+
+## Description
+
+An ExploitCatalogVulnerabilityCategorization is TODO
```
Comment (jeff-schutt) [inline model/Security/Classes/ExploitCatalogVulnerabilityCategorization.md:11]: [https://www.cisa.gov/known-exploited-vulnerabilities-catalog](https://www.cisa.gov/known-exploited-vulnerabilities-catalog)

**Review by jeff-schutt — CHANGES_REQUESTED** [review]
Added links to relevant specifications and/or documentation

**Comment by armintaenzertng** [comment]
Thanks for your suggestions, @jeff-schutt! I added them in.

**Comment by goneall** [comment]
@tsteenbe @jeff-schutt - with the suggestions merged in, do you want to go ahead and merge this PR so we can use it as a starting point for any additional review/changes?

**Review by goneall — APPROVED** [review]
Since this is going into the security profile branch and not the main branch, I suggest we go ahead and merge it so we can use it as a base to fill in the missing "TODO"'s.  Once merged, we should create issues from some of the comments (e.g. the property is used in multiple locations).

**Pull request merged**